gdraw/gscrollbar.c

```diff
@@ -520,8 +520,7 @@ static void GScrollBarFit(GScrollBar *gsb) {
     minheight = 2*(gsb->thumbborder+gsb->arrowsize) + GDrawPointsToPixels(gsb->g.base,2);
 
     if ( gsb->g.vert ) {
-	if ( gsb->g.r.width==0 )
-	    gsb->g.r.width = GDrawPointsToPixels(gsb->g.base,_GScrollBar_Width);
+	gsb->g.r.width = GDrawPointsToPixels(gsb->g.base,_GScrollBar_Width);
```

Comment:
I looked at this a bit more. _GScrollBar_Width is a global variable that gets initialized once, so the call to GDrawPointsToPixels would return the same thing every time unless the screen resolution were to change or unless _GScrollBar_Width were used before and after initialization. If you see inconsistent scrollbar widths, there is probably a deeper problem. Can you provide more information about the observed problem?